src/Wsdl/Event/RequestPostEvent.php

```diff
@@ -0,0 +1,101 @@
+<?php
+declare(strict_types = 1);
+
+namespace Skaut\Skautis\Wsdl\Event;
+
+use Serializable;
+use stdClass;
+
+class RequestPostEvent implements Serializable
```

Comment:
Potřebujeme tady implementovat Serializable? Vidím, že se všechny properties standardně serializují.

Reply:
Není to vyloženě potřeba. Já preferuji explicitnost spíš než minimální počet řádků. Ale pokud ti to vyloženě vadí, tak to odstraním, zas tolik na tom netrvám. Nepamatuji si totiž proč se to původně implementovalo. Myslím že to dělalo problém v nějakém debug baru v Nette/Symfony.